### PR TITLE
Added huber_loss - Updated pull request

### PR DESCRIPTION
**Pull request by ZiadAmerr**
Closes https://github.com/unifyai/ivy/issues/7065

### CODE REVIEW

**Comment by ZiadAmerr** [comment]
@kurshakuz Here you can find the updated pull request

File: ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py [hunk 1]
```diff
@@ -274,6 +274,49 @@ def test_torch_smooth_l1_loss(
...
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        allow_inf=False,
+        shared_dtype=True,
+    ),
+    delta=helpers.floats(min_value=0, max_value=5),
+    reduction=st.sampled_from(["none", "mean", "sum"]),
+)
+def test_huber_loss(
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py:289]:
```suggestion
def test_torch_huber_loss(
```

**Review by kurshakuz — CHANGES_REQUESTED** [review]
Thanks! Now it is passing the test. Please fix the change I requested and I will merge your PR 🙂

**Comment by ZiadAmerr** [comment]
Done

**Review by kurshakuz — APPROVED** [review]
Thanks for your efforts @ZiadAmerr ! LGTM!

**Pull request merged**